CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ This change log adheres to standards from [Keep a CHANGELOG](http://keepachangel
 
 ## [Unreleased]
 
+
+## [1.10.1] - 2016-07-02
```

Comment:
Yes, it is a new feature but not for the public API exposed by the `eslint-plugin-import` package which is what SemVer applies to in this instance.

> On 3. juli 2016, at 17:10, Jordan Harband <notifications@github.com> wrote:
> 
> In CHANGELOG.md:
> 
> > @@ -5,6 +5,11 @@ This change log adheres to standards from [Keep a CHANGELOG](http://keepachangel
> >  
> >  ## [Unreleased]
> >  
> > +
> > +## [1.10.1] - 2016-07-02
> That is a new feature.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>